client/packages/requisitions/src/ResponseRequisition/ListView/CreateRequisitionModal.tsx

```diff
@@ -81,7 +77,7 @@ export const CreateRequisitionModal: FC<CreateRequisitionModalProps> = ({
 
   return (
     <Modal
-      height={height}
```

Comment:
Daaangg yea this is tricky. I think maybe better to revert and open an issue, as now it'll not be responsive at all :(. And to be fair, preexisting issue, your PR didn't introduce this!

Reply:
I think it's fine not to have it responsive for now? Most the other modals aren't responsive from what I can see.. Only the createRequisitionModal and PatientSearchModals are responsive

Reply:
Sweet as